src/libraries/System.Numerics.Tensors/ref/System.Numerics.Tensors.netcore.cs

```diff
@@ -157,4 +416,127 @@ public static partial class TensorPrimitives
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        [System.ObsoleteAttribute("GetHashCode() on TensorSpan will always throw an exception.")]
+#pragma warning disable CS0809 // Obsolete member overrides non-obsolete member
+        public override int GetHashCode() { throw null; }
+#pragma warning restore CS0809 // Obsolete member overrides non-obsolete member
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public ref T GetPinnableReference() { throw null; }
+        public static bool operator ==(System.Numerics.Tensors.TensorSpan<T> left, System.Numerics.Tensors.TensorSpan<T> right) { throw null; }
+        public static implicit operator System.Numerics.Tensors.ReadOnlyTensorSpan<T> (System.Numerics.Tensors.TensorSpan<T> span) { throw null; }
+        public static implicit operator System.Numerics.Tensors.TensorSpan<T> (T[]? array) { throw null; }
+        public static bool operator !=(System.Numerics.Tensors.TensorSpan<T> left, System.Numerics.Tensors.TensorSpan<T> right) { throw null; }
+        public System.Numerics.Tensors.TensorSpan<T> Slice(scoped System.ReadOnlySpan<System.Buffers.NIndex> indexes) { throw null; }
+        public System.Numerics.Tensors.TensorSpan<T> Slice(scoped System.ReadOnlySpan<System.Buffers.NRange> ranges) { throw null; }
+        public override string ToString() { throw null; }
+        public bool TryCopyTo(System.Numerics.Tensors.TensorSpan<T> destination) { throw null; }
+        public bool TryFlattenTo(System.Span<T> destination) { throw null; }
+        public ref partial struct Enumerator
+        {
+            private object _dummy;
+            private int _dummyPrimitive;
+            public ref T Current { get { throw null; } }
+            public bool MoveNext() { throw null; }
+        }
+    }
+    public sealed partial class Tensor<T> : System.Collections.Generic.IEnumerable<T>, System.Collections.IEnumerable, System.Numerics.Tensors.IReadOnlyTensor<System.Numerics.Tensors.Tensor<T>, T>, System.Numerics.Tensors.ITensor<System.Numerics.Tensors.Tensor<T>, T>
+    {
+        internal Tensor() { }
+        public static System.Numerics.Tensors.Tensor<T> Empty { get { throw null; } }
+        public nint FlattenedLength { get { throw null; } }
+        public bool IsEmpty { get { throw null; } }
+        public bool IsPinned { get { throw null; } }
+        public bool IsReadOnly { get { throw null; } }
+        public System.Numerics.Tensors.Tensor<T> this[System.Numerics.Tensors.Tensor<bool> filter] { get { throw null; } }
+        public ref T this[scoped System.ReadOnlySpan<System.Buffers.NIndex> indexes] { get { throw null; } }
+        public System.Numerics.Tensors.Tensor<T> this[scoped System.ReadOnlySpan<System.Buffers.NRange> ranges] { get { throw null; } set { } }
```

Comment:
This one shouldn't need `scoped`, its a reference type and can't capture the span ever already

Reply:
Same for many of the other inputs in this type (I don't think its required for the interface implementation either, as it should be implicit here given the `this` can't have ref fields and doesn't return something that is `ref` or could have a ref field)